src/modules/Auth/ducks/authLogout.js

```diff
@@ -0,0 +1,48 @@
+import Duck from 'services/Duck';
+import { apiPaths, paths, actionTypes } from 'modules/Auth/const';
+import AuthService from 'services/Auth';
+import actionsGlobal from 'actions';
+import { isAuthModulePath } from 'modules/Auth/utils';
+import { push as goToPage } from 'react-router-redux';
+import URI from 'urijs';
+
+const actionCoreName = 'AU_AUTH_LOGOUT';
+
+const authLogoutDuck = new Duck(
+  {
+    name: actionCoreName,
+    urlBuilder: apiPaths.logout,
+  },
+);
+
+function recievePrincipal(principal) {
```

Comment:
If it is called only for principal clearing, then I would call it smth like "clearPrincipal" and remove call param